dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/client/Client.java

```diff
@@ -36,6 +40,12 @@ public ProviderAuthenticationResult continueFlow(HttpServletRequest request, Htt
         Flow flow = AbstractFlow.getInstance(oidcClientConfig);
         return flow.continueFlow(request, response);
     }
+    
+    public boolean validate(TokenResponse tokenResponse) throws TokenValidationException {
```

Comment:
For the future, we can make this method "void" since tokenResponseValidator.validate(tokenResponse); should always throw an exception when failing to validate the token(s).

Reply:
sounds good Teddy. Thanks